README.md

```diff
@@ -611,25 +587,59 @@ they don't already exist:
+  automatically, with [versioning](https://docs.aws.amazon.com/AmazonS3/latest/dev/Versioning.html), 
+  [server-side encryption](https://docs.aws.amazon.com/AmazonS3/latest/dev/UsingServerSideEncryption.html), and 
+  [access logging](https://docs.aws.amazon.com/AmazonS3/latest/dev/ServerLogs.html) enabled.
 
   In addition, you can let terragrunt tag the bucket with custom tags that you specify in
-  `remote_state.config.s3_bucket_tags`. See sample configuration in section
-  [Filling in remote state settings with Terragrunt](#filling-in-remote-state-settings-with-terragrunt).
+  `remote_state.config.s3_bucket_tags`.
 
 * **DynamoDB table**: If you are using the [S3 backend](https://www.terraform.io/docs/backends/types/s3.html) for
   remote state storage and you specify a `dynamodb_table` (a [DynamoDB table used for
   locking](https://www.terraform.io/docs/backends/types/s3.html#dynamodb_table)) in `remote_state.config`, if that table
-  doesn't already exist, Terragrunt will create it automatically, with [server-side encryption](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/EncryptionAtRest.html) enabled, including a primary key called `LockID`.
+  doesn't already exist, Terragrunt will create it automatically, with 
+  [server-side encryption](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/EncryptionAtRest.html) 
+  enabled, including a primary key called `LockID`.
 
   In addition, you can let terragrunt tag the DynamoDB table with custom tags that you specify in
-  `remote_state.config.dynamodb_table_tags`. See sample configuration in section
-  [Filling in remote state settings with Terragrunt](#filling-in-remote-state-settings-with-terragrunt).
+  `remote_state.config.dynamodb_table_tags`.
 
 **Note**: If you specify a `profile` key in `remote_state.config`, Terragrunt will automatically use this AWS profile
 when creating the S3 bucket or DynamoDB table.
 
 
+#### S3-specific remote state settings
+
+For the `s3` backend, the following config options can be used for S3-compatible object stores, as necessary:
+
+```hcl
+remote_state {
+  # ...
+
+  skip_bucket_versioning         = true # use only if the object store does not support versioning
+  skip_bucket_ssencryption       = true # use only if non-encrypted Terraform State is required and/or the object store does not support server-side encryption
+  skip_bucket_accesslogging      = true # use only if the cost for the extra object space is undesirable or the object store does not support access logging
+  enable_lock_table_ssencryption = true # use only if non-encrypted DynamoDB Lock Table for the Terraform State is required and/or the NoSQL database service does not support server-side encryption
+
+  shared_credentials_file     = "/path/to/credentials/file"
+  skip_region_validation      = true
+  skip_credentials_validation = true
+  skip_requesting_account_id  = true
+  skip_get_ec2_platforms      = true
```

Comment:
skip_get_ec2_platforms and skip_requesting_account_id are deprecated and should be removed.

Reply:
More info on this?

Reply:
https://www.terraform.io/docs/backends/types/s3.html#skip_get_ec2_platforms
https://www.terraform.io/docs/backends/types/s3.html#skip_region_validation
https://www.terraform.io/docs/backends/types/s3.html#skip_requesting_account_id

Reply:
Thanks! https://github.com/gruntwork-io/terragrunt/pull/731/commits/866e689a5562822a3a4d6f8b081c2ec34110b41a